gradle.properties

```diff
@@ -7,12 +7,9 @@ org.gradle.daemon=false
 minecraft_version=1.18.1
 forge_version=39.0.5
 
-# Mappings
-mappings_channel=official
-mappings_version=1.18.1
-# https://github.com/ParchmentMC/Parchment
-#mappings_channel=parchment
-#mappings_version=1.17.1-2021.12.05
+# Mappings https://github.com/ParchmentMC/Parchment

```

Comment:
Why are we changing to parchment from official?

Reply:
Parchment is official variables/classes/methods with the addition of community made javadoc and parameter names. It's just nicer to know why a random function takes 5 int values and what they do instead of needing to look at the implementation